operator/gitops/argocd/pipeline-service/pipelines-as-code/kustomization.yaml

```diff
@@ -6,17 +6,17 @@ commonLabels:
 
 resources:
   - allow-argocd.yaml
-  - https://github.com/openshift-pipelines/pipelines-as-code/releases/download/v0.16.0/release.yaml
+  - https://github.com/openshift-pipelines/pipelines-as-code/releases/download/v0.17.3/release.yaml
 
 # TODO backing off until https://issues.redhat.com/browse/SRVKP-3134 is resolved
 # images:
-#  - name: ghcr.io/openshift-pipelines/pipelines-as-code-controller:v0.16.0
+#  - name: ghcr.io/openshift-pipelines/pipelines-as-code-controller:v0.17.3
 #    newName: registry.redhat.io/openshift-pipelines/pipelines-pipelines-as-code-rhel8
 #    newTag: v1.10.2-1
```

Comment:
update this and the other similar refs to v1.11, as that is what is required to get to 0.17.x

Reply:
never mind per http://dashboard.apps.cicd.ospqa.com/releases/componentmatrix/ 1.10 already is at 0.17.x